wordpress/wp-content/plugins/memberful-wp/src/acl/post_options.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+function memberful_wp_get_post_available_to_any_registered_users( $post_id ) {
+  return get_post_meta( $post_id, 'memberful_available_to_any_registered_user', TRUE ) === "1";
+}
+
+function memberful_wp_set_post_available_to_any_registered_users( $post_id, $is_viewable_by_any_registered_users ) {
```

Comment:
This and `memberful_wp_set_post_available_to_anybody_subscribed_to_a_plan()` are pretty similar, but I don't see a nice abstraction to solve the code duplication.